static/js/containers/ChannelPage.js

```diff
@@ -55,6 +56,7 @@ class ChannelPage extends React.Component {
               <ChannelSidebar channel={channel} />
             </Card>
           </div>
+          <br className="clear"/>
```

Comment:
I am open to any solutions you can come up with, although this is in my opinion, a basic failure with the CSS model and contained floats inside divs. But I am open to any way you'd prefer to do this. I do want those divs actually contained inside the container div though.